contracts/finance/VestingWalletConfidentialFactory.sol

```diff
@@ -15,33 +15,19 @@ import {VestingWalletCliffConfidential} from "./VestingWalletCliffConfidential.s
  */
 abstract contract VestingWalletConfidentialFactory {
     struct VestingPlan {
-        address beneficiary;
         externalEuint64 encryptedAmount;
-        uint48 startTimestamp;
-        uint48 durationSeconds;
-        uint48 cliffSeconds;
+        bytes initialization;
```

Comment:
Very minor, maybe we can call it `args` or `initArgs` to be consistent with what we have in `Clones` lib. (`initialization` let me think it is a full encoded function call). Feel free to drop that point if you like the current wording.

Reply:
good idea